IonDotnet/IonDotnet.csproj

```diff
@@ -5,12 +5,15 @@
     <RootNamespace>IonDotnet</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <LangVersion>7.3</LangVersion>
-    <PackageId>IonDotnet</PackageId>
-    <Version>0.0.3-alpha</Version>
-    <Authors>hdhoang</Authors>
+    <PackageId>AmazonIonDotnet</PackageId>
+    <Version>1.0.0</Version>
```

Comment:
Feels optimistic for a preview release unless we're confident the API is stable.  Perhaps `0.4.0` or `0.9.0`?

Reply:
I'm pretty confident in that since API fixes were a major focus, but there's no harm in going with 0.9.0.